eZ/Publish/Core/Persistence/Legacy/Content/Mapper.php

```diff
@@ -430,10 +432,11 @@ protected function extractFieldValueFromRow(array $row, $type)
      * Creates CreateStruct from $content.
      *
      * @param \eZ\Publish\SPI\Persistence\Content $content
+     * @param bool $preserveMainLanguage
      *
      * @return \eZ\Publish\SPI\Persistence\Content\CreateStruct
      */
-    public function createCreateStructFromContent(Content $content)
+    public function createCreateStructFromContent(Content $content, $preserveMainLanguage = false)
```

Comment:
Said in another way: this method is only used from copy, so there is no need to add this additional complexity and having to maintain it and understand it later.